content/language-agnostic/encyclopedia/what-is-a-class.md

```diff
@@ -0,0 +1,34 @@
+<!--
+---
+Subjects:
+  - "What is a Class?"
+Tags: 
+  - "language-agnostic"
```

Comment:
Should be taken from https://github.com/Codecademy/codepedia-content/blob/main/documentation/tags.md